client/blocks/author-compact-profile/style.scss

```diff
@@ -0,0 +1,64 @@
+.author-compact-profile {
+	font-size: 14px;
+
+	.gravatar {
+    	display: flex;
+    	margin: auto;
+    }
+
+    .follow-button {
+    	border: 0;
+    	border-radius: 0;
+    	padding: 0;
+
+    	.gridicon__follow {
+    		fill: $blue-medium;
+    	}
+
+    	.follow-button__label {
+    		color: $blue-medium;
+
+    		@include breakpoint( "<660px" ) {
+    			display: inline-block;
+    		}
+    	}
+
+		&:hover {
+
+			.gridicon__follow {
+				fill: lighten( $gray, 10% );
+			}
+
+			.follow-button__label {
+				color: lighten( $gray, 10% );
+			}
+		}
+    }
+}
+
+.author-compact-profile .external-link,
+.author-compact-profile__site-link,
+.author-compact-profile__follow {
+    align-items: center;
+    color: $blue-medium;
+	display: flex;
```

Comment:
That was weird, it didn't appear indented in my editor but I think re-tabbing them in https://github.com/Automattic/wp-calypso/pull/7127/commits/9ec68fa2f41ddf394e04a9f70f8d899cf28ed3e3 worked.

![screenshot 2016-08-02 12 12 00](https://cloud.githubusercontent.com/assets/4924246/17341669/8b78fb06-58aa-11e6-89ba-2e7d94322872.png)